docs/README.md

```diff
@@ -0,0 +1,13 @@
+## Building Docs Locally
+
+```bash
+python setup.py install (#or ./build.sh) 
+cd docs
+mkdir -p ../build/docs
```

Comment:
The shell script, if it completes, [shows a clickable link](https://github.com/facebookresearch/habitat-sim/blob/28a24acfd10c1c262c88a795a5c5b62bae2195c8/docs/build.sh#L37-L41) with output location (which I thought could be intuitive enough), but having to do this explicit step makes it work less than ideal. I'll follow up with a PR that removes the need to explicitly create an output directory.

Reply:
@mosra -- is this ready to merge?

Reply:
You're right, this fell through cracks and I completely forgot about it. The README isn't wrong on its own, so let's merge this.

> I'll follow up with a PR that removes the need to explicitly create an output directory.

This still holds, of course.